lib/fika/env.ex

```diff
@@ -28,7 +28,9 @@ defmodule Fika.Env do
         "kernel./(Float,Float)" => "Float",
         "kernel.|(Bool,Bool)" => "Bool",
         "kernel.&(Bool,Bool)" => "Bool",
-        "kernel.!(Bool)" => "Bool"
+        "kernel.!(Bool)" => "Bool",
+        "kernel.-(Int)" => "Int",
+        "kernel.+(Int)" => "Int"
```

Comment:
I think we can remove this + too?

Reply:
Sure, forgot about it.
I’ll fix that asap